src/Header/SetCookie.php

```diff
@@ -337,7 +337,8 @@ public function getFieldValue()
         }
 
         $sameSite = $this->getSameSite();
-        if ($sameSite !== null && in_array($sameSite, self::SAME_SITE_ALLOWED_VALUES, true)) {
+        if ($sameSite !== null
+            && in_array(strtolower($sameSite), array_map('strtolower', self::SAME_SITE_ALLOWED_VALUES), true)) {
```

Comment:
I would suggest to change the values in the constant `SAME_SITE_ALLOWED_VALUES` as it is marked as `internal`. Then we don't need to lowercase it here.